<launch>
	<rosparam command="load" file="$(find puma_joy)/param/joy_params.yaml" />

	<node
		pkg="puma_joy"
		type="puma_joy_node.py"
		name="puma_joy"
		output="screen"
	>
	</node>
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node" respawn_delay="2" >
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
		<param name="default_trig_val" value="true" />
	</node>

</launch>